tests/controller_tests/conftest.py

```diff
@@ -9,4 +9,5 @@
 @pytest.fixture(scope="function")
 def example_file() -> BinaryIO:
     """An example spreadsheet in towns-fund schema format."""
-    return (resources / "DLUCH_Data_Model_V3.4_EXAMPLE.xlsx").open("rb")
+    with open(resources / "DLUCH_Data_Model_V3.4_EXAMPLE.xlsx", "rb") as file:
```

Comment:
nice. Should we swap the other tests that open files to use context manager too? (in test_ingest). Not really part of this PR, but could add here anyway? Or just backlog it as a minor fix and do it later?

Reply:
happy to add it now 👍